ui/lib/pki/addon/templates/overview.hbs

```diff
@@ -19,7 +19,49 @@
 </Toolbar>
 
 {{#if this.model.hasConfig}}
-  {{! TODO show overview items }}
+  <div
+    class="selectable-card-container has-grid has-top-margin-l has-two-col-grid
+      {{if (or (eq this.model.issuers 403) (eq this.model.roles 403)) 'has-three-col-grid' 'has-two-col-grid'}}"
+  >
+    {{#if this.issuersCanList}}
+      <SelectableCard
+        @cardTitle="Issuers"
+        @total={{if (eq this.model.issuers 404) 0 this.model.issuers.length}}
+        @subText="The total number of issuers in this PKI mount. Includes both root and intermediate certificates."
+        @actionText="View issuers"
+        @actionTo="issuers"
+      />
+    {{/if}}
+    {{#if this.rolesCanList}}
```

Comment:
If `this.rolesCanList` ends up being the only controller attribute, perhaps we can remove the `setupController` method all together and change this to 
```suggestion
    {{#if (not (eq this.model.roles 403))}}
```